classes/Http/Action/Reviewer/Talk/ViewAction.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Http\Action\Reviewer\Talk;
+
+use OpenCFP\Domain\Talk;
+use Symfony\Component\HttpFoundation;
+use Symfony\Component\Routing;
+use Twig_Environment;
+
+final class ViewAction
+{
+    /**
+     * @var Talk\TalkHandler
+     */
+    private $talkHandler;
+
+    /**
+     * @var Twig_Environment
+     */
+    private $twig;
+
+    /**
+     * @var Routing\Generator\UrlGeneratorInterface
+     */
+    private $urlGenerator;
+
+    public function __construct(
+        Talk\TalkHandler $talkHandler,
+        Twig_Environment $twig,
+        Routing\Generator\UrlGeneratorInterface $urlGenerator
+    ) {
+        $this->talkHandler  = $talkHandler;
+        $this->twig         = $twig;
+        $this->urlGenerator = $urlGenerator;
+    }
+
+    public function __invoke(HttpFoundation\Request $request): HttpFoundation\Response
```

Comment:
Is there a reason for not using ```@Template``` here?

Reply:
Yes, the reason is that we would then have to remove the type declaration, as 

* for a redirect, we would need to return an instance of `HttpFoundation\Response`
* for the success case, we would need to return an array